scripts/push_docker.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -e
```

Comment:
Similar to above, I am not quite sure why we need `-e` here given the handling through `die` et al.